zebrad/src/components/tracing/component.rs

```diff
@@ -97,7 +97,7 @@ impl<A: abscissa_core::Application> Component<A> for Tracing {
 
     fn before_shutdown(&self, _kind: Shutdown) -> Result<(), FrameworkError> {
         if let Some(ref grapher) = self.flamegrapher {
-            tracing::info!("writing flamegraph");
+            debug!("writing flamegraph");
```

Comment:
The user specifically requested this action, so it's important they can see it happening. (Particularly if they can't find the flamegraph file.)